### DISCUSSION TITLE
A wrong translation to "assets" in Chinese for GitHub app

### DISCUSSION THREAD

**Discussion by xiaopangju** [community/community#4006] [Mobile] [answered]
![IMG_20210522_220157.JPG](https://user-images.githubusercontent.com/56964531/119229192-594a9980-bb49-11eb-9f69-ff60d14124db.JPG)
Please look at the image.
You will see the wrong translation if you go to one repo has a release page and see the end of the page.

The right translation should be "资源", not "资产".

Related discussion:https://github.com/github/feedback/discussions/74

**Comment by elisealix22** [comment, marked as answer]
Hi @xiaopangju , thank you so much for reporting this! I filed an issue with our localization team internally. If you see any other bugs or have any other requests, please let us know! We really appreciate it.

**Reply by xiaopangju** [reply]
Is there a way to contact the Simplified Chinese localization team directly? For example, email or website. I think it is better to use Chinese to describe translation errors than to use English to describe translation errors.

**Reply by xiaopangju** [reply]
@elisealix22 can you reply to my comment?
